homeassistant/components/comelit/quality_scale.yaml

```diff
@@ -0,0 +1,76 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: no actions
+  appropriate-polling: done
+  brands: done
+  common-modules: done
```

Comment:
there are a few things that could be moved to an base ComelitEntity class, like:
- `_has_entity_name`
- unique id (adding the config-entry-id part)
- device info

and maybe the `_api` and `_device` attributes too.

But it is not much, so I don't think it is mandatory here.